SU-SWS/stanford_profile_helper
# D8CORE-5418: hot fix adding back padding for mobile view

# READY FOR REVIEW

# Summary
- Adding back the padding for the Basic+ cards when in responsive view of md max width
- 

# Review By (Date)
- When does this need to be reviewed by?

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Urgency
- How urgent is this? (Normal, High)

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Pull amptesting-test so you have content 
4. Navigate to http://[YOUR LOCAL]/interior-page/list-paragraphs/list-card-display-full-width
4. Verify there is spacing below the Basic+ Cards on all views. This adds it back when the view is Medium or smaller.
Examples:
<img width="469" alt="Screen Shot 2022-02-03 at 10 56 07 AM" src="https://user-images.githubusercontent.com/1187335/152410065-1b19017d-01c8-48c8-9a97-6b297e353489.png">
<img width="474" alt="Screen Shot 2022-02-03 at 10 55 55 AM" src="https://user-images.githubusercontent.com/1187335/152410077-4bb249ae-73bd-4323-8754-29057ce3bbfc.png">
<img width="290" alt="Screen Shot 2022-02-03 at 10 55 31 AM" src="https://user-images.githubusercontent.com/1187335/152410078-93198aac-7aee-403b-b321-f04399011aba.png">



### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? NO

## Front End Validation

## Backend / Functional Validation
### Code

### Code security

## General

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [D8CORE-5418](https://stanfordits.atlassian.net/browse/D8CORE-5418)
- https://github.com/SU-SWS/stanford_profile_helper/pull/165

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)
